tidb-cloud/tidb-cloud-quickstart.md

```diff
@@ -133,37 +129,65 @@ We provide Capital Bikeshare sample data for you to easily import data and run s
 
     If your TiDB cluster is hosted by AWS (the Developer Tier is hosted by AWS by default), fill in the following parameters:
 
-    - **Data Source Type**: `AWS S3`.
-    - **Bucket URL**: enter the sample data URL `s3://tidbcloud-samples/data-ingestion/`.
-    - **Data Format**: select **TiDB Dumpling**.
-    - **Setup Credentials**: enter `arn:aws:iam::385595570414:role/import-sample-access` for Role-ARN.
-    - **Target Cluster**: fill in the **Username** and **Password** fields.
-    - **DB/Tables Filter**: leave this field blank.
+    - **Data Format**: select **TiDB Dumpling**
+    - **Location**: `AWS`
+    - **Bucket URL**: enter the sample data URL `s3://tidbcloud-samples/data-ingestion/`
+    - **Role-ARN**: enter `arn:aws:iam::385595570414:role/import-sample-access`
+    - **Target Cluster**: fill in the **Username** and **Password** fields
+
+    Confirm the compliance of cross region. Click **Next**.
 
     </div>
 
     <div label="GCP">
 
     If your TiDB cluster is hosted by GCP, fill in the following parameters:
 
-    - **Data Source Type**: `Google Cloud Stroage`.
-    - **Bucket URL**: enter the sample data URL `gcs://tidbcloud-samples-us-west1`.
     - **Data Format**: select **TiDB Dumpling**.
+    - **Location**: `Google Cloud`
+    - **Bucket URL**: enter the sample data URL `gcs://tidbcloud-samples-us-west1`.
     - **Target Cluster**: fill in the **Username** and **Password** fields.
-    - **DB/Tables Filter**: leave this field blank.
+
+    Confirm the compliance of cross region. Click **Next**.
 
     </div>
     </SimpleTab>
 
-4. Click **Import**.
+4. Modify the file patterns and add the table filter rules if needed. Click **Next**.
```

Comment:
Since this doc is a quick start and users do not know the file structure of the sample data, can we tell users to click **Next** directly?

Reply:
OK deleted detailed instructions.